### PR TITLE
GTM-update

### PR DESCRIPTION
**Pull request by AmaanISE**
GTM fixes

### CODE REVIEW

File: src/Context/GtmContext.php [hunk 1]
```diff
@@ -64,6 +64,9 @@ public function dataLayerSettingShouldBe($key, $value) {
    */
   public function getDataLayerSettingShouldMatch($key, $regex) {
     $property_value = $this->getDataLayerValue($key);
+    if (is_bool($property_value)) {
+        $property_value = ($property_value==true) ?'true':'false';
```
Comment (marcelovani) [inline src/Context/GtmContext.php]: Please format the syntax with spaces, you can use PHP storm for that via menu Code / Reformat code

Comment (AmaanISE) [reply src/Context/GtmContext.php]: updated

File: src/Context/GtmContext.php [hunk 2]
```diff
@@ -64,6 +64,9 @@ public function dataLayerSettingShouldBe($key, $value) {
    */
   public function getDataLayerSettingShouldMatch($key, $regex) {
     $property_value = $this->getDataLayerValue($key);
+    if (is_bool($property_value)) {
+      $property_value = ($property_value==true) ?'true':'false';
```
Comment (marcelovani) [inline src/Context/GtmContext.php]: Still missing spaces after `?` and around `:`

Comment (AmaanISE) [reply src/Context/GtmContext.php]: updated

**Pull request merged**